charts/synthetics-minion/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: synthetics-minion
 description: New Relic Synthetics Containerized Private Minion (CPM)
 type: application
-version: 1.0.34
+version: 1.0.35
```

Comment:
We'll need to bump this to `1.0.36`.

Reply:
Strangely it's showing that the last version is `1.0.34` from this diff, but we already have `1.0.35` merged in. Perhaps your fork just needs to be synced up again.

Reply:
bumped version

Reply:
the versions keep changing faster than this pr can get merged